db/migrate/20200910204740_create_users.rb

```diff
@@ -0,0 +1,16 @@
+class CreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :users do |t|
+      t.string :uid
+      t.string :email
+      t.string :access_token
+      t.string :refresh_token
+      t.string :first_name
+      t.string :last_name
+      t.string :birthdate
```

Comment:
I'm curious about why this is a string instead of a date
And since it's a string, how are we going to normalize the input type? ie, make sure that all user birthdates are entered into the form and saved in our table in the same format

Reply:
You're right, it should be a datetime object.  Michael and I thought of making it a string due to difficulties that we thought could arise with the date aspect, but now I think that those concerns were unfounded.  This is an easy fix.

Reply:
To clarify: we aren't using the birthdate for much of anything (I don't think?)

Reply:
Resolution: specify format in form input instructions and check input before saving to make sure it's in correct format